src/ai/backend/storage/api/manager.py

```diff
@@ -485,7 +485,7 @@ async def rename_file(request: web.Request) -> web.Response:
                 t.Key("vfid"): tx.UUID(),
                 t.Key("relpath"): tx.PurePath(relative_only=True),
                 t.Key("new_name"): t.String(),
-                t.Key("is_dir"): t.ToBool(),  # ignored since 22.03
```

Comment:
You don't have to remove parenthesis of `t.ToBool()` because Trafaret automatically handles the difference of type reference and instance reference when there is no argument of a trafaret class.